src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -76,6 +76,7 @@
 import java.util.function.BiPredicate;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
+import sun.nio.fs.AbstractFileSystemProvider;
```

Comment:
Minor nit, you can probably group this with the imports of the other sun.nio classes.